yt_dlp/extractor/generic.py

```diff
@@ -2748,6 +2763,7 @@ def filter_video(urls):
 
         entries = []
         for video_url in orderedSet(found):
+            video_url = video_url.encode().decode('unicode-escape')
```

Comment:
should this be part of `unescapeHTML`?

Reply:
it could be, but really it's cleaning up javascript instead of cleaning up HTML

Reply:
Let's leave it like this for now. If we find other instances of this issue, we can consider moving into utils